src/route/nodes/netid/nodeEdit/NodeEdit.tsx

```diff
@@ -123,221 +121,275 @@ export const NodeEdit: React.FC<{
-            label={String(t('node.localaddress'))}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            defaultValue={node.name}
-            name={'name'}
-            label={String(t('node.name'))}
-          />
-        </Grid>
-
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            disabled
-            defaultValue={t('node.publickey')}
-            name={'publickey'}
-            label={String(t('node.publickey'))}
-          />
-        </Grid>
-
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            defaultValue={node.postup}
-            name={'postup'}
-            label={String(t('node.postup'))}
-            disabled={!serverConfig.RCE}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            defaultValue={node.postdown}
-            label={String(t('node.postdown'))}
-            name={'postdown'}
-            disabled={!serverConfig.RCE}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            defaultValue={node.allowedips ? node.allowedips.join(',') : ''}
-            label={String(t('node.allowedips'))}
-            name={'allowedips'}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            defaultValue={node.persistentkeepalive}
-            label={String(t('node.persistentkeepalive'))}
-            name={'persistentkeepalive'}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <NmFormInputText
-            disabled={!node.isrelay}
-            defaultValue={node.relayaddrs ? node.relayaddrs.join(',') : ''}
-            label={String(t('node.relayaddrs'))}
-            name={'relayaddrs'}
-          />
-        </Grid>
-        <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
-          <LocalizationProvider dateAdapter={AdapterDateFns}>
-            <DateTimePicker
-              renderInput={(props) => (
-                <TextField
-                  fullWidth={false}
-                  sx={{ maxWidth: '15rem' }}
-                  {...props}
-                />
-              )}
-              label={String(t('node.expdatetime'))}
-              value={
-                expTime
-                  ? datePickerConverter(expTime)
-                  : datePickerConverter(node.expdatetime)
+        <Tooltip title={String(t('helper.dynamicendpoint'))} placement="top">
+          <Grid item xs={6} sm={4} md={3} sx={rowMargin}>
+            <NmFormInputSwitch
+              label={String(t('node.isstatic'))}
+              name={'isstatic'}
+              defaultValue={isIPDynamic}
+            />
+          </Grid>
+        </Tooltip>
+        <Tooltip title={String(t('helper.defaultlistenport'))} placement="top">
```

Comment:
the tooltips should go around the NmFormInputTexts components.. if the fields are disabled, put a <span></span> around the input text first..

Reply:
done